test/test_basic/faucet.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
 	},
 
 	plugins: {
-		"nunjucks": {
+		nunjucks: {
```

Comment:
delete this repo

Reply:
😘

Reply:
![Dr. Evil airquoting "nunjucks"](https://i.imgflip.com/37fcdo.jpg)

Reply:
🍪